lib/swtpmtest.pm

```diff
@@ -88,12 +90,12 @@ sub swtpm_verify {
 
     # Check the vm guest is up via listening to the port 22
     assert_script_run("wget --quiet " . data_url("swtpm/ssh_port_chk_script") . " -P $image_path");
-    assert_script_run("bash $image_path/ssh_port_chk_script", timeout => 200);
+    assert_script_run("bash $image_path/ssh_port_chk_script", timeout => 210);
 
     # Login to the vm and run the commands to check tpm device
     my $user = 'root';
-    my $passwd = $testapi::password;
-    my $ip_addr = script_output("ip n | awk '/192\\.168/ {print \$1}'");
+    my $passwd = 'nots3cr3t';
```

Comment:
Can I know the reason why you use the plain password here?

Reply:
In JeOS, the system password is set to `linux`, so we can not use `$testapi::password;`.